report/src/presence_technologies/bluetooth.tex

```diff
@@ -0,0 +1,44 @@
+\section{Bluetooth} \label{sec:bluetooth}
+
+Another possible technology for presence detection is Bluetooth.
+Bluetooth is a common technology used in many devices such as smartphones, tablets, and laptops.
+Because it is a standardized technology, all devices that support Bluetooth can communicate with each other.
+
+There are two different types of radio signals used in Bluetooth: Classic and Low Energy (LE).
+They differ in terms of power consumption, range, and data transfer rate.\cite{BluetoothSIG}
+
+\begin{figure}[h!]
+  \centering
+  \includegraphics[width=1\textwidth]{Bluetooth_Technology_Overview_Graphic.png}
+  \caption{Overview of the Bluetooth Classic and Low Energy technologies\cite{BluetoothOverview}.}
+  \label{fig:classic_vs_LE}
+\end{figure}
+
+\subsection{Bluetooth Classic} \label{sec:bluetooth_classic}
+The Bluetooth Classic radio is the original version of the Bluetooth technology introduced in 1999 for point-to-point device communication.
+It is used for data transfer between devices but is primarily used for wireless audio streaming across devices because of its high data transfer rate.
+According to the Bluetooth Special Interest Group (SIG), the data transfer rate of Bluetooth Classic is 3 Mb/s.
```

Comment:
Bluetooth Special Interest Group cite, preferably on the page where they write it.

Reply:
Added that cite